Core/NoOpRunner.Core/Shapes/PowerUp.cs

```diff
@@ -0,0 +1,66 @@
+﻿using NoOpRunner.Core.Entities;
+using NoOpRunner.Core.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NoOpRunner.Core.Shapes
+{
+    public class PowerUp : BaseShape
+    {
+        // Player 1 PowerUp | Player 2 PowerUp
+        Dictionary<string, string> powerUps = new Dictionary<string, string>(){
+            { "speed_boost", "rocket" },
```

Comment:
This dictionary doesn't seem to make a lot of sense, unless these powerups for each player are tied in some way in spawning logic, which doesn't seem to be the case.